src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/MsQuicListener.cs

```diff
@@ -157,7 +159,7 @@ private void Dispose(bool disposing)
                 return;
             }
 
-            Stop();
+            Stop().GetAwaiter().GetResult();
```

Comment:
This is absolutely terrible solution and we need to come up with better one. I'm taking this into account for Listener API.

Reply:
And this too, can we leave a TODO comment?.. Just so it will not slip from our attention

Reply:
Done